### PR TITLE
Rename fixes

### PR DESCRIPTION
**Pull request by 31453**
package.json moloch -> arkime
cookies/headers moloch -> arkime
molochError -> serverError

### CODE REVIEW

File: viewer/viewer.js [hunk 1]
```diff
@@ -212,16 +212,16 @@ if (Config.get('passwordSecret')) {
     }
 
     // S2S Auth
-    if (req.headers['x-moloch-auth']) {
-      const obj = Config.auth2obj(req.headers['x-moloch-auth'], false);
+    if (req.headers['x-arkime-auth']) {
```
Comment (awick) [inline viewer/viewer.js:215]: Can you make this accept both the x-moloch-auth and x-arkime-auth, same with line below

File: viewer/viewerUtils.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = (Config, Db, molochparser, internals) => {
     if (!options.headers) {
       options.headers = {};
     }
-    options.headers['x-moloch-auth'] = Config.obj2auth({
+    options.headers['x-arkime-auth'] = Config.obj2auth({
```
Comment (awick) [inline viewer/viewerUtils.js:14]: set x-moloch-auth for now

**Review by awick — CHANGES_REQUESTED** [review]

File: viewer/viewer.js (hunk 1 above)
Comment (31453) [reply viewer/viewer.js]: 👍

**Comment by awick** [comment]
LGTM

**Pull request merged**